internal/controller/api_vms_portforward.go

```diff
@@ -30,23 +32,40 @@ func (controller *Controller) portForwardVM(ctx *gin.Context) responder.Responde
 		return responder.Code(http.StatusBadRequest)
 	}
 
+	waitRaw := ctx.Query("wait")
+	wait, err := strconv.ParseUint(waitRaw, 10, 16)
```

Comment:
Perhaps we should check the error here? Similarly to the code a few lines above.

Reply:
Good catch! I wonder why linter didn't catch it. Fixed in [69ab796](https://github.com/cirruslabs/orchard/pull/65/commits/69ab796870b3e4664e4ece3db0b0397770726aa8)